SwiftCards/HomePageViewController.swift

```diff
@@ -0,0 +1,50 @@
+import UIKit
+import MultipeerConnectivity
+
+class HomePageViewController: UIViewController {
+    var advertiserAssistant: MCAdvertiserAssistant!
+    var session: MCSession!
+    var peerID: MCPeerID!
+    var viewController: ViewController!
+    var gameViewController: GameViewController!
+    override func viewDidLoad() {
```

Comment:
Hopefully we can refactor this later to make it slightly smaller - I bet some of these lines are no longer needed after your changes